src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,55 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "cinemaHalls")
+public class CinemaHall {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private int capacity;
+    private String description;
+
+    public CinemaHall(int capacity, String description) {
```

Comment:
Don't create a default constructor when it is not needed. If the no-argument constructor is not used in the application, it's better to remove it to avoid creating instances of CinemaHall without setting the required fields.